website/addons/citations/provider.py

```diff
@@ -17,6 +17,14 @@ def __init__(self, provider_name):
     def serializer(self):
         pass
 
+    def check_credentials(self, node_addon):
+        valid = True
+        try:
+            node_addon.api.client()
```

Comment:
Is this correct? `client` appears to be a property of the `Mendeley` provider class, not a method.